core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -482,6 +490,9 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
         |                              maven repo, then maven central and any additional remote
         |                              repositories given by --repositories. The format for the
         |                              coordinates should be groupId:artifactId:version.
+        |  --packages-exclude          Comma-separated list of groupId:artifactId to exclude while
```

Comment:
The option reads a little funny. Maybe `--exclude-packages`? Not a big deal though.